src/encoder.rs

```diff
@@ -1609,7 +1609,7 @@ fn encode_partition_bottomup<T: Pixel, W: Writer>(
       let w: &mut W = if cw.bc.cdef_coded {w_post_cdef} else {w_pre_cdef};
       let tell = w.tell_frac();
       cw.write_partition(w, tile_bo, PartitionType::PARTITION_NONE, bsize);
-      (w.tell_frac() - tell) as f64 * fi.lambda / ((1 << OD_BITRES) as f64)
+      compute_rd_cost(fi, tell, 0)
```

Comment:
Shouldn't this be `w.tell_frac() - tell` too instead of just `tell`?

Reply:
Yup, as my awcy run shows ;_;